components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -324,6 +327,7 @@ class GeckoEngineSession(
                 notifyObservers {
                     onProgress(PROGRESS_STOP)
                     onLoadingStateChange(false)
+                    onLoadComplete(actualURL)
```

Comment:
GeckoView already reports the URL as we expect it. Adding this new behavior here obscures the GV behavior. Especially if the URL may have changed between onPageStart() and onPageStop() then we'd report the wrong one here. I think it would be good to avoid adding logic here.

Reply:
👍